content/en/serverless/installation/nodejs.md

```diff
@@ -228,17 +221,10 @@ If you are deploying your Lambda function as a container image, you cannot use t
 npm install --save datadog-lambda-js dd-trace
```

Comment:
I know this isn't something you changed, but I don't think `--save` is a real flag for `npm install` 🤔